### ISSUE TITLE
Unknown method coords

### ISSUE THREAD

**Issue by mjago** [acook/remedy#9]
Hi
remedy is looking promising!
However the viewport example fails with unknown method coords for Remedy::Size:(0x0) with gem 0.1.0

> viewport.rb:26:in `range_find': undefined method `coords' for #<Remedy::Size:(0x0)> (NoMethodError)

```ruby
include Remedy
  joke = Content.new
  joke << "Q: What's the difference between a duck?"
  joke << "A: Purple, because ice cream has no bones!"

  screen = Viewport.new
  screen.draw joke
```

**Comment by acook** [comment]
Thanks for the bug report! I'm looking into it now.

**Comment by acook** [comment]
Sorry about that. Looks like the `coords` methods is a leftover from a previous iteration of `Size` and was totally unnecessary.

I've also added a test to ensure that the example in the readme runs, so if there's a regression or a change in the API it will show up when running the tests.

Fixed in this commit: https://github.com/acook/remedy/commit/d553b1389d5f828d45df06510c28ac8536cbe449

Thanks again for the bug report!
